core/citrus-base/src/main/java/org/citrusframework/functions/core/RandomStringFunction.java

```diff
@@ -18,21 +18,23 @@
 
 import java.util.List;
 import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
 
 import org.citrusframework.context.TestContext;
 import org.citrusframework.exceptions.InvalidFunctionUsageException;
 import org.citrusframework.functions.Function;
 
 import static java.lang.Boolean.parseBoolean;
 import static java.lang.Integer.parseInt;
+import static java.lang.System.currentTimeMillis;
 
 /**
  * Function generating a random string containing alphabetic characters. Arguments specify
  * upper and lower case mode.
  *
  */
 public class RandomStringFunction implements Function {
-    private static Random generator = new Random(System.currentTimeMillis());
+    private static final Random generator = new Random(currentTimeMillis());
```

Comment:
```suggestion
    private static final Random RANDOM = new Random(currentTimeMillis());
```

convention for `static final` variables.

Reply:
It's not really a constant for which i chose lower case. . You can always set a new seed. I'd keep it in lower case but I have no really strong opinion on that. I applied the name suggestion though.

Reply:
I see the thought of that. it's constant in terms of "one runtime", right? neither have I a strong opinion, in that case.